Makefile

```diff
@@ -89,7 +89,7 @@ container:
 
 	docker build --pull -t $(PREFIX)/heapster-$(ARCH):$(VERSION) $(TEMP_DIR)
 ifneq ($(OVERRIDE_IMAGE_NAME),)
-	docker tag -f $(PREFIX)/heapster-$(ARCH):$(VERSION) $(OVERRIDE_IMAGE_NAME)
+	docker tag $(PREFIX)/heapster-$(ARCH):$(VERSION) $(OVERRIDE_IMAGE_NAME)
```

Comment:
FYI, this broke e2e tests
```
docker tag gcr.io/google_containers/heapster-amd64:v1.3.0-beta.1 heapster:e2e_test
Error response from daemon: Conflict: Tag heapster:e2e_test is already set to image 1b32c9c98284, if you want to replace it, please use -f option
Makefile:68: recipe for target 'container' failed
make[1]: *** [container] Error 1
make[1]: Leaving directory '/go/src/k8s.io/heapster
```
https://k8s-gubernator.appspot.com/build/kubernetes-jenkins/pr-logs/pull/heapster/1572/pull-heapster-e2e/238/

I'm going to revert this change (only Makefile part).